AndroidAnnotations/androidannotations/src/main/java/org/androidannotations/handler/FragmentArgHandler.java

```diff
@@ -113,5 +113,12 @@ private void createBuilderInjectionMethod(Element element, EFragmentHolder holde
 		JVar arg = method.param(paramClass, fieldName);
 		method.body().invoke(builderArgsField, bundleHelper.getMethodNameToSave()).arg(argKeyStaticField).arg(arg);
 		method.body()._return(_this());
+
+		String docComment = processingEnv.getElementUtils().getDocComment(element);
+		if (docComment != null) {
+			method.javadoc().append(docComment.replaceAll("\r", "").trim());
```

Comment:
You mean you removed the `\r`, right? But why is this needed? CodeModel assumes only `\n` line ending and break with `\r\n`?